test/eventlistener_test.go

```diff
@@ -284,11 +285,29 @@ func TestEventListenerCreate(t *testing.T) {
 		t.Fatalf("Error creating POST request: %s", err)
 	}
 	req.Header.Add("Content-Type", "application/json")
-	_, err = http.DefaultClient.Do(req)
+	resp, err := http.DefaultClient.Do(req)
 	if err != nil {
 		t.Fatalf("Error sending POST request: %s", err)
 	}
 
+	if resp.StatusCode > http.StatusAccepted {
+		t.Errorf("sink did not return 2xx response. Got status code: %d", resp.StatusCode)
+	}
+	wantBody := sink.Response{
+		EventListener: "my-eventlistener",
+		Namespace:     namespace,
+	}
+	var gotBody sink.Response
+	if err := json.NewDecoder(resp.Body).Decode(&gotBody); err != nil {
+		t.Fatalf("failed to read/decode sink response: %v", err)
+	}
+	if diff := cmp.Diff(wantBody, gotBody, cmpopts.IgnoreFields(sink.Response{}, "EventID")); diff != "" {
+		t.Errorf("unexpected sink response -want/+got: %s", diff)
+	}
+	if gotBody.EventID == "" {
+		t.Errorf("sink response no eventID")
+	}
+
```

Comment:
Thanks for adding this!! :tada: 

i think this test is getting super long - even more so with our changes queued up in #279 XD 

Maybe in a future PR we can start breaking this up into multiple functions .